lib/cli-runner/runner.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import Mocha from 'mocha';
```

Comment:
Have you considered using [`node:test#run`](https://nodejs.org/api/test.html#runoptions)?

Since this is builtin to Node.js, this saves a dependency. It also provides nice interop with alternative test frameworks like `uvu` or `tape`.

Reply:
I have, but we also have [@vscode/test-web](https://www.npmjs.com/package/@vscode/test-web) and it would be nice if tests were isomorphic, when applicable, between browsers and Electron. Browsers won't have `node:test` available.